examples/go-kprobe-counter/main.go

```diff
@@ -31,7 +31,7 @@ type Stats struct {
 	Counter uint64
 }
 
-//go:generate bpf2go -cc clang -no-strip -cflags "-O2 -g -Wall" bpf ./bpf/kprobe_counter.c -- -I.:/usr/include/bpf:/usr/include/linux
+//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -cc clang -no-strip -cflags "-O2 -g -Wall" bpf ./bpf/kprobe_counter.c -- -I.:/usr/include/bpf:/usr/include/linux
```

Comment:
Since we don't need to install bpf2go any more, please remove it from https://bpfman.io/main/getting-started/example-bpf/#building-locally

Reply:
This probably needs more refactoring:
1. remove installing bpf2go installation instruction.
2. regenerating the eBPF bytecode normally isn't necessary, as it already comes with the examples. I would like to suggest moving this into its own section about "Tinkering with Examples".
3. building the examples also isn't necessary, as `go run ...` takes care of building on-the-fly.

IMHO this would simplify the examples user journey, at least for the local host. Shall I move forward on this?

Reply:
I agree with everything except Bullet 2. eBPF bytecode doesn't come with the examples, unless I'm missing something. We used to check it into git, but that was removed several months ago.